lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -47,6 +58,36 @@ def release_languages(self, languages):
             enabled=True
         ).save()
 
+    def create_user(self):
+        """
+        Creates the user log in
+        """
+        # Create one user and save it to the database
+
+        self.user = UserFactory.build(username='test', email=self.email)
+        self.user.set_password(self.pwd)
+        self.user.save()
+
+        # Create a registration for the user
+        RegistrationFactory(user=self.user)
+
+        # Create a profile for the user
+        UserProfileFactory(user=self.user)
+
+    def user_login(self):
+        """
+        Log the user in
+        """
+        # Create the test client
+        self.client = Client()
+
+        # Get the login url & log in our user
+        try:
+            login_url = reverse('login_post')
+        except NoReverseMatch:
+            login_url = reverse('login')
+        self.client.post(login_url, {'email': self.email, 'password': self.pwd})
```

Comment:
The standard way we login in unit tests (that don't need to test the login endpoints themselves) is by simply doing:
`self.client.login(username=self.user.username, password=self.pwd)`